services.yml

```diff
@@ -43,6 +43,7 @@ services:
       MYSQL_HOST: "${MYSQL_HOST}"
       RABBIT_HOST: "${RABBIT_HOST}"
       REDIS_HOST: "${REDIS_HOST}"
+      _JAVA_OPTIONS: "-Djava.net.preferIPv4Stack=true"
```

Comment:
Does `java.net.preferIPv4Stack: true` work?

Reply:
Yes, It did.